text/0023-readme.md

```diff
@@ -0,0 +1,68 @@
+# Module Documentation
+- Start Date: 2020/04/17
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/23
+
+## Decision, including impact on distributions
+OpenMRS ESMs will maintain documentation of their features, APIs,
+configuration schema, and compatibility concerns in their README.
+
+The OpenMRS Wiki page
+[List of frontend modules](https://wiki.openmrs.org/pages/viewpage.action?pageId=224527568)
+will link directly to the README in the ESM's GitHub repo's page. The
```

Comment:
Can we make it "will link to the GitHub repository of the ESM"?

Reply:
Yep! Thanks for reviewing!